tkp/db/database.py

```diff
@@ -91,11 +90,13 @@ def __init__(self, **kwargs):
                                              self.host,
                                              self.port,
                                              self.database),
-                                            echo=False
+                                            echo=False,
+                                            poolclass=NullPool,
                                             )
         self.Session = sessionmaker(bind=self.alchemy_engine)
```

Comment:
how about instead of `self.Session` ->  `self.session_factory`, to distinguish it from `sqlalchemy.orm.session.Session` (the SqlAlchemy docs are terrible for confusing names like this also, drives me up the wall a bit).